Documentation/concepts/networking/fragmentation.rst

```diff
@@ -0,0 +1,22 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _concepts_fragmentation:
+
+IP fragment handling (beta)
+===========================
```

Comment:
```suggestion
IPv4 fragment handling (beta)
=============================
```

It's IPv4 only (IPv6 handles this differently), so it would be worth mentioning `IPv4` at least once, be it in the title or elsewhere, I think.